From: "Christoph M. Becker" <cmbecker69@gmx.de>
Date: Wed, 22 Jan 2020 12:32:46 +0100
Subject: Fix MSVC build

MSVC does only support the `inline` keyword for C++, so we define it to
`__inline`.
---
 ext/mbstring/libmbfl/filters/mbfilter_big5.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/ext/mbstring/libmbfl/filters/mbfilter_big5.c b/ext/mbstring/libmbfl/filters/mbfilter_big5.c
index e04d81d..47813d4 100644
--- a/ext/mbstring/libmbfl/filters/mbfilter_big5.c
+++ b/ext/mbstring/libmbfl/filters/mbfilter_big5.c
@@ -138,6 +138,10 @@ static unsigned short cp950_pua_tbl[][4] = {
 	{0xf70f,0xf848,0xc740,0xc8fe},
 };
 
+#ifdef PHP_WIN32
+# define inline __inline
+#endif
+
 static inline int is_in_cp950_pua(int c1, int c) {
 	if ((c1 >= 0xfa && c1 <= 0xfe) || (c1 >= 0x8e && c1 <= 0xa0) ||
 			(c1 >= 0x81 && c1 <= 0x8d) || (c1 >= 0xc7 && c1 <= 0xc8)) {
